provider/barebones-go/.goreleaser.yml

```diff
@@ -0,0 +1,32 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - arm64
+      - amd64
+    goarm:
+      - "7"
+    ignore:
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: arm64
```

Comment:
sad windows noises

Reply:
This should still be building windows x86....i just realized I didnt add it to the par release tho, ill add that